fileStore/commands.py

```diff
@@ -8,7 +8,7 @@
 #TODO: Add documentation
 
 
-def save_file_base(spec, file_path, custom=None):
+def save_file_base(spec, file_path, custom=None, collection_version=0):
```

Comment:
The internals have access to `FileBase`.  Ophyd shouldn't need to know those details, but I can see use cases where you might want it.